src/main/resources/ivylogviewer.fxml

```diff
@@ -11,37 +11,62 @@
-            <MenuItem fx:id="menuPointAbout" mnemonicParsing="false" text="About IvyLogViewer" />
-          </items>
-        </Menu>
-      </menus>
-    </MenuBar>
-    <AnchorPane fx:id="treeAnchorPane" maxHeight="-1.0" maxWidth="-1.0" prefHeight="445.0" prefWidth="640.0" VBox.vgrow="ALWAYS">
-      <children>
-      	<TreeView fx:id="logTreeView" layoutY="44.0" prefHeight="302.0" prefWidth="640.0" AnchorPane.bottomAnchor="24.0" AnchorPane.leftAnchor="0.0" AnchorPane.rightAnchor="0.0" AnchorPane.topAnchor="44.0" />
-            <ComboBox fx:id="minimalLevel" layoutX="108.0" layoutY="10.0" prefWidth="150.0" />
-            <Label layoutX="14.0" layoutY="14.0" text="Show Level:" />
-            <Button fx:id="searchButton" defaultButton="true" layoutX="574.0" layoutY="10.0" mnemonicParsing="false" text="Search" AnchorPane.rightAnchor="14.0" />
-            <TextField fx:id="searchField" layoutX="418.0" layoutY="10.0" prefHeight="25.0" prefWidth="228.0" AnchorPane.rightAnchor="74.0" />
-            <Label fx:id="filepathLabel" layoutX="14.0" layoutY="87.0" prefHeight="17.0" prefWidth="611.0" AnchorPane.bottomAnchor="5.0" AnchorPane.leftAnchor="5.0" AnchorPane.rightAnchor="5.0" />
-      </children>
-    </AnchorPane>
-  </children>
+<VBox prefHeight="400.0" prefWidth="800.0" stylesheets="@ivylogviewer.css"
+	xmlns="http://javafx.com/javafx/8.0.151" xmlns:fx="http://javafx.com/fxml/1"
+	fx:controller="com.axonivy.ivy.supplements.logviewer.MainController">
+	<children>
+		<MenuBar fx:id="menuBar" VBox.vgrow="NEVER">
+			<menus>
+				<Menu mnemonicParsing="false" text="File">
+					<items>
+						<MenuItem fx:id="menuPointOpen" mnemonicParsing="false"
+							text="Open…" />
+						<MenuItem fx:id="menuPointCopy" mnemonicParsing="false"
+							text="Copy" />
+						<SeparatorMenuItem mnemonicParsing="false" />
+						<MenuItem fx:id="menuPointCollapse" mnemonicParsing="false"
+							text="Collapse All" />
+						<SeparatorMenuItem mnemonicParsing="false" />
+						<MenuItem fx:id="menuPointQuit" mnemonicParsing="false"
+							text="Quit" />
+					</items>
+				</Menu>
+				<Menu mnemonicParsing="false" text="Help">
+					<items>
+						<MenuItem fx:id="menuPointAbout" mnemonicParsing="false"
+							text="About IvyLogViewer" />
+					</items>
+				</Menu>
+			</menus>
+		</MenuBar>
+		<AnchorPane fx:id="treeAnchorPane" maxHeight="-1.0"
+			maxWidth="-1.0" prefHeight="445.0" prefWidth="640.0" VBox.vgrow="ALWAYS">
+			<children>
+				<TreeView fx:id="logTreeView" layoutY="44.0" prefHeight="302.0"
+					prefWidth="640.0" AnchorPane.bottomAnchor="24.0"
+					AnchorPane.leftAnchor="0.0" AnchorPane.rightAnchor="0.0"
+					AnchorPane.topAnchor="44.0" />
+				<ComboBox fx:id="minimalLevel" layoutX="100.0" layoutY="10.0"
+					prefWidth="100.0" />
+				<Label layoutX="14.0" layoutY="14.0" text="Show Level:" />
+				<Button fx:id="applyNewFilter" defaultButton="true" layoutX="280.0"
+					layoutY="10.0" mnemonicParsing="false" text="+" />
```

Comment:
Please use a more appealing button icon... On my linux OS it's halfway overlapped by the startTimeField btw

Reply:
@ivy-dhu I'm aware of the overlap. However, I think the "+" is the most implicit icon to use here. What icon would you prefer?

Reply:
I would actually even prefer a time picker slider. Then you would not even have to have a button to apply the filter. If the slider gets too big you can also put it on a second line. Also it would be very helpful when the max and min values would already be written in inputs next to the slider, so you wouldn't have to write the whole time-syntax by hand.

Reply:
We've had the discussion RCR-internal about the timepickslider before I started the implementation of this feature and came to the conclusion that its really impractical and just looks really ugly.

Imho its hard to filter the exact/desired (mili)seconds with a slider, so I'd definitely not recommend this. 
cc: @ultimatemanu

Reply:
That's okay for me then, but please set the max and min time in the boxes at least.
Also what's exactly the problem with the overlapping button?
Have you tried to integrate fontawesomefx for cool icons?

Reply:
The overlapping button was fixed with my latest commit. I didnt integrate fa to keep it consitant with the icons that we're already there. 

I've just implemented the setting of max and min time, but imo its confusing for the user because it kinda implies that the miliseconds are needed aswell for the filtering.

Reply:
You're right about the milliseconds. Best is to just write the normal time there (HH:MM) since the logviewer in the real world will be used in other ways than we are expecting now, e.g. it will have a log of several hours and then you won't care about seconds.
Btw I don't get any icon on the button in the current build...

Reply:
Seconds are definitely important, there could be thousands of entries during 60 seconds so this is mandatory imo. 

Icon should be fine, works local for me and the file was successfully commited to the repository.